proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -172,4 +166,46 @@ Since new parameters are added, Core, iOS, Java Suite, RPC, Protocol, and HMI ar
 
 ## Alternatives considered
 
-Nothing.  
+The flow shown in Proposed solution is complicated, so the conditions have been arranged.
+
+### Change the app registration flow
+
+![registeringApp_flow2](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow2.png)
+
+1. The developer will set their `requiresAudioSupport` to true.
+2. The library will send a `StartService` for the RPC service with a new param in the payload, `requiresAudioSupport`.
+3. Core will receive the `StartService` for the RPC service:
+    i. If `requiresAudioSupport` was set to false or not set, it will continue in flow.
+    ii. Core will check its current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
+    iii. If there is no Audio support, Core will check if it supports the auto connect BT function. If it does it will continue in flow.
+    iv. If `requiresAudioSupport` is true and there is no supported audio methods and Core does not support the auto connect BT function, it will send a `StartServiceNAK` with the reason `No audio support available`.
```

Comment:
Please add line breaks to fix the formatting, so each item in the list is on its own line.

Reply:
Added line breaks.